src/install/resolution.zig

```diff
@@ -370,7 +370,7 @@ pub fn ResolutionType(comptime SemverIntType: type) type {
                     lhs_string_buf,
                     rhs_string_buf,
                 ),
-                else => unreachable,
+                else => true,
```

Comment:
```suggestion
```

I would remove this case altogether to force the user to have to explicitly specify which dependency does what. Slightly less fragile, in general